packages/components/src/organisms/RatingSummary/RatingDistribution.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef, type HTMLAttributes } from 'react'
+import RatingDistributionItem from './RatingDistributionItem'
+
+export interface RatingDistributionProps
+  extends HTMLAttributes<HTMLOListElement> {
+  /**
+   * The rating distribution
+   */
+  distribution: {
+    starsOne: number
+    starsTwo: number
+    starsThree: number
+    starsFour: number
+    starsFive: number
+  }
+  /**
+   * Optional test ID for testing.
+   */
```

Comment:
just to keep the pattern
```suggestion
 /**
   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
   */
```